src/installer/pkg/sfx/Microsoft.NETCore.App/PackageOverrides.txt

```diff
@@ -1,91 +1,97 @@
+runtime.opensuse.13.2-x64.runtime.native.System|4.3.1
+runtime.opensuse.13.2-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.opensuse.13.2-x64.runtime.native.System.Net.Http|4.3.1
+runtime.opensuse.13.2-x64.runtime.native.System.Net.Security|4.3.1
+runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.opensuse.13.2-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.opensuse.42.1-x64.runtime.native.System|4.3.1
+runtime.opensuse.42.1-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.opensuse.42.1-x64.runtime.native.System.Net.Http|4.3.1
+runtime.opensuse.42.1-x64.runtime.native.System.Net.Security|4.3.1
+runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.opensuse.42.1-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.osx.10.10-x64.runtime.native.System|4.3.1
+runtime.osx.10.10-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.osx.10.10-x64.runtime.native.System.Net.Http|4.3.1
+runtime.osx.10.10-x64.runtime.native.System.Net.Security|4.3.1
+runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.Apple|4.3.1
+runtime.osx.10.10-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.rhel.7-x64.runtime.native.System|4.3.1
+runtime.rhel.7-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.rhel.7-x64.runtime.native.System.Net.Http|4.3.1
+runtime.rhel.7-x64.runtime.native.System.Net.Security|4.3.1
+runtime.rhel.7-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.rhel.7-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.ubuntu.14.04-x64.runtime.native.System|4.3.1
+runtime.ubuntu.14.04-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.ubuntu.14.04-x64.runtime.native.System.Net.Http|4.3.1
+runtime.ubuntu.14.04-x64.runtime.native.System.Net.Security|4.3.1
+runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.ubuntu.14.04-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.ubuntu.16.04-x64.runtime.native.System|4.3.1
+runtime.ubuntu.16.04-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.ubuntu.16.04-x64.runtime.native.System.Net.Http|4.3.1
+runtime.ubuntu.16.04-x64.runtime.native.System.Net.Security|4.3.1
+runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.ubuntu.16.04-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
+runtime.ubuntu.16.10-x64.runtime.native.System|4.3.1
+runtime.ubuntu.16.10-x64.runtime.native.System.IO.Compression|4.3.2
+runtime.ubuntu.16.10-x64.runtime.native.System.Net.Http|4.3.1
+runtime.ubuntu.16.10-x64.runtime.native.System.Net.Security|4.3.1
+runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography|4.3.4
+runtime.ubuntu.16.10-x64.runtime.native.System.Security.Cryptography.OpenSsl|4.3.3
 System.AppContext|4.3.0
-System.Buffers|4.4.0
+System.Buffers|4.5.1
```

Comment:
Rather than listing exact serviced versions like this, what if we instead list a higher version that will never need to update?   We know that this package will never win over the framework copy (unless we ship a much newer version) so why bother churning on these serviced versions?

I care less about stuff we no longer build, but for things we're still building like System.Buffers it might be better to just list a threshold; `4.6.0`, or `5.0.0`, or even `9.0.0`.